CourseApp.Tests/TaskTest.cs

```diff
@@ -0,0 +1,20 @@
+namespace CourseApp.Tests
+{
+    using Xunit;
+    using static System.Math;
+
+    public class TaskTest
+    {
+        [Theory]
+        [InlineData(1.25, 1.748)]
+        [InlineData(3.25, 3.016)]
+        [InlineData(1.84, 2.038)]
+        [InlineData(3.81, 3.25)]
+        [InlineData(5.62, 3.834)]
+        public void TestCalc(double x, double expected)
```

Comment:
Where are the tests for task A and task B?

there you may simply check sizes of returned arrays